app/src/main/contrib/components/Responsibilities/Overview/ResponsibilityOverviewDescription.tsx

```diff
@@ -8,30 +8,73 @@ interface ResponsibilityOverviewDescriptionProps {
+                           Download demographic data for Nigeria
+                        </li>
+                        <li>
+                            Download csv templates for central and stochastic burden estimates, and for underlying
+                            parameter values
+                        </li>
+                        <li>
+                            Upload central burden estimates for each scenario, and review any problems the system has
+                            detected in the uploaded data
+                        </li>
+                        <li>
+                            Upload your parameters fike
+                        </li>
+                    </ul>
+                    Useful links:
+                    <ul>
+                        <li>
+                            <a
+                                href={outputGuide}
+                                target="_blank">Output specification guidance (PDF)</a>
+                        </li>
+                    </ul>
+                    If you have any questions or anything is not as you expected, please email&nbsp;
+                    <a href={`mailto:${settings.vimcEmail}`}>
+                        {settings.vimcEmail}
+                    </a>
+                </div>;
+            }
+            else {
+                return <div>
+                    On this page you can:
+                    <ul>
+                        <li>
+                            Access the standardised demographic data (Most of the demographic data is based on the UN
+                            World Population Prospects (UNWPP) 2017 Revision).
+                        </li>
+                        <li>
+                            Download the burden estimate templates for your disease area.
+                        </li>
+                        <li>
+                            Submit the burden estimates generated by your model for one country of choice.
+                        </li>
+                    </ul>
+                    Output specifications guidance is available on the VIMC website (<a
+                    href="https://www.vaccineimpact.org/2017-12-19-request-for-proposals-yellow-fever-rubella-Japanese-encephalitis-closing-date-January-30/"
+                    target="_blank">
+                    https://www.vaccineimpact.org/2017-12-19-request-for-proposals-yellow-fever-rubella-Japanese-encephalitis-closing-date-January-30/</a>).
 
-                If you have any questions or anything is not as you expected, please email&nbsp;
-                <a href={`mailto:${settings.vimcEmail}`}>
-                    {settings.vimcEmail}
-                </a>
-            </div>;
+                    If you have any questions or anything is not as you expected, please email&nbsp;
+                    <a href={`mailto:${settings.vimcEmail}`}>
+                        {settings.vimcEmail}
+                    </a>
+                </div>;
+            }
```

Comment:
It might be nice to break these two different bits of guidance into separate components (probably overkill) or just separate functions. So the logic can look more like "if special touchstone { renderSpecialTouchstoneGuidance() } else { renderNormalRFPGuidance() }, which will be easier to read I think.